### PR TITLE
[TACHYON-1384] fix TFS test with CDH

### PR DESCRIPTION
**Pull request by siyangy**
https://tachyon.atlassian.net/browse/TACHYON-1384

http://stackoverflow.com/questions/32660836/java-lang-linkageerror-loader-constraint-violation-in-interface-itable-initiali

### CODE REVIEW

**Comment by siyangy** [comment]
verified by running 
mvn test -Dtest=tachyon.hadoop.TFSTest -DfailIfNoTests=false  -Dhadoop.version=2.6.0-cdh5.4.8

File: clients/unshaded/src/test/java/tachyon/hadoop/TFSTest.java [hunk 1]
```diff
@@ -49,6 +50,7 @@
  */
 @RunWith(PowerMockRunner.class)
 @PrepareForTest({FileSystemContext.class, FileSystemMasterClient.class, UserGroupInformation.class})
+@PowerMockIgnore("javax.security.*")
```
Comment (calvinjia) [inline clients/unshaded/src/test/java/tachyon/hadoop/TFSTest.java]: Could you add a comment explaining why we are ignoring this?

**Comment by aaudiber** [comment]
LGTM

**Comment by calvinjia** [comment]
LGTM

**Pull request merged**